### ISSUE TITLE
TypeError: strftime() argument 1 must be str, not None

### ISSUE THREAD

**Issue by danillos** [mremallin/DailyOrganizer#1]
When opening the Daily Note
```
Traceback (most recent call last):
  File "/Applications/Sublime Text.app/Contents/MacOS/Lib/python33/sublime_plugin.py", line 1458, in run_
    return self.run()
  File "/Users/danillos/Library/Application Support/Sublime Text/Installed Packages/DailyOrganizer.sublime-package/DailyOrganizer.py", line 65, in run
  File "/Users/danillos/Library/Application Support/Sublime Text/Installed Packages/DailyOrganizer.sublime-package/DailyOrganizer.py", line 61, in get_todays_note_file_path
  File "/Users/danillos/Library/Application Support/Sublime Text/Installed Packages/DailyOrganizer.sublime-package/DailyOrganizer.py", line 48, in get_current_folder
TypeError: strftime() argument 1 must be str, not None
```

**Comment by mremallin** [comment]
Thanks for the report! Would you be able to post your `DailyOrganizer.sublime-settings` configuration file?

**Comment by danillos** [comment]
> Thanks for the report! Would you be able to post your `DailyOrganizer.sublime-settings` configuration file?

```
{
    // folder_structure defines the folder structure to store notes
    // Available format strings are listed here: https://docs.python.org/3.6/library/datetime.html#strftime-strptime-behavior
    "folder_structure": "~/DailyOrganizer/%Y/%m/%d",
    // Name of the main notes file stored in each folder
    // Extensions other than txt can also be used and will automatically
    // use the right sytax highlighter if installed (ex. Today.md for Markdown)
    "main_notes_file_name": "Today.txt",
}

```

**Comment by mremallin** [comment]
Thank you Danillo! This is now fixed in v0.0.3.
